content/topics/web-frontend/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 _db_id: 76
 content_type: topic
-ready: false
 title: Web Frontend
----
```

Comment:
This content doesn't get turned into a card on the learner's boards so it doesn't matter if it is marked as ready or not.

We need to make sure all the content that gets displayed on user's boards is ready for them.

So if a syllabus links to something directly or indirectly then it should be made to be ready.